### ISSUE TITLE
cmap bug

### ISSUE THREAD

**Issue by idealvin** [vrapper/vrapper#668]
Hi, I have add this line to my vrapperrc file
```sh
cmap >c %s/\<case\>/  case/g
```

However, when I type :>c in eclipse, It show the following: 
```sh
:%s/\\\/  case/g
```

In vim, It should be `:%s/\<case\>/  case/g`,    
I have upgrade vrapper to the newest unstable version, the bug is still there.   
Please fix it, Thanks!

**Comment by albertdev** [comment]
@idealvin 
I didn't find the time yet to test your problem but you closed it yourself. Does that mean that there is no problem or did you simply find a workaround?

**Comment by idealvin** [comment]
@albertdev 

Yeah, I need to indent C++ case statements manually. I finally found vrapper's `regular expression grammar` is different from vim's. In Vim, \< \> is used to match exactly a word. Vrapper doesn't support it. I have solved it by using vrapper's regex style.   

Another problem is that vrapper does not support vim's `abbreviate` command. As a C++ programmer, I use `#i` as the abbreviatation of `#include`. I have solved it with the inoremap which mapping `#i` into `#include`. So everything is OK now!
